docs/deploy-event-router.md

```diff
@@ -0,0 +1,111 @@
+will enable event collection equivalent to the same capability from 3.11 releases or earlier.
+Currently, all events are logged to `STDOUT`, collected by the logging collector and indexed to the `.operations` index when 
+the `EventRouter` is deployed into an operations namespace (e.g. `openshift-*`). Elevated permissions are required to create the cluster level roles and bindings referenced in the following template:
+
+```
+kind: Template
+apiVersion: v1
+metadata:
+  name: eventrouter-template
+  annotations:
+    description: "A pod forwarding kubernetes events to cluster logging stack."
+    tags: "events,EFK,logging, cluster-logging"
+objects:
+  - kind: ServiceAccount
+    apiVersion: v1
+    metadata:
+      name: cluster-logging-eventrouter
+      namespace: ${NAMESPACE}
+  - kind: ClusterRole
+    apiVersion: v1
+    metadata:
+      name: event-reader
+    rules:
+    - apiGroups: [""]
+      resources: ["events"]
+      verbs: ["get", "watch", "list"]
+  - kind: ClusterRoleBinding
+    apiVersion: v1
+    metadata:
+      name: event-reader-binding
+    subjects:
+    - kind: ServiceAccount
+      name: cluster-logging-eventrouter
+      namespace: ${NAMESPACE}
+    roleRef:
+      kind: ClusterRole
+      name: event-reader
+  - kind: ConfigMap
+    apiVersion: v1
+    metadata:
+      name: cluster-logging-eventrouter
+      namespace: ${NAMESPACE}
+    data:
+      config.json: |-
+        {
+          "sink": "stdout"
+        }
+  - kind: Deployment
+    apiVersion: apps/v1
+    metadata:
+      name: cluster-logging-eventrouter
+      namespace: ${NAMESPACE}
+      labels:
+        component: eventrouter
+        logging-infra: eventrouter
+        provider: openshift
+    spec:
+      selector:
+		matchLabels:
+          component: eventrouter
+          logging-infra: eventrouter
+          provider: openshift
+      replicas: "1"
+      template:
+        metadata:
+          labels:
+            component: eventrouter
+            logging-infra: eventrouter
+            provider: openshift
+          name: cluster-logging-eventrouter
+        spec:
+          serviceAccount: cluster-logging-eventrouter
+          containers:
+            - name: kube-eventrouter
+              image: ${IMAGE}
+              imagePullPolicy: IfNotPresent
+              resources:
+                limits:
+                  memory: ${MEMORY}
+                requests:
+                  cpu: ${CPU}
+                  memory: ${MEMORY}
+              volumeMounts:
+              - name: config-volume
+                mountPath: /etc/eventrouter
+          volumes:
+            - name: config-volume
+              configMap:
+                name: cluster-logging-eventrouter
+parameters:
+  - name: IMAGE
+    displayName: Image
+    value: "quay.io/openshift/origin-logging-eventrouter:latest"
+  - name: MEMORY
+    displayName: Memory
+    value: "128Mi"
+  - name: CPU
+    displayName: CPU
+    value: "100m"
+  - name: NAMESPACE
```

Comment:
I'm not sure we should give users the ability to choose the namespace since they won't have the ability to change it when CLO performs this

Reply:
I'm indifferent here.  I don't THINK there should be a reason to move it elsewhere, but it goes inline with my comment regarding the SA name.  If we peg one I would be onboard with doing the same for the namespace.

Reply:
Although not directly related to the topic, may I ask that eventrouter used to be deployed to the default namespace; why it's being changed?
https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_logging_eventrouter
> eventrouter is deployed to default project, has a service account and its own role to read events

Reply:
Originally this was because the events deployed to any other namespace but infra would cause the events to go to that index.  In 4.x, 'openshift-*' are all considered infra so the logs will end up in the operations namespace.  The correct solution should be to add a custom parser for events that sends them to an index associated with that namespace.

Reply:
Thank you, @jcantrill!  I.e., to support eventrouter in the Collector Piplelines, the custom parser for events is going to be added?